MANIFEST.in

```diff
@@ -4,3 +4,6 @@ prune tests
 include README.md
 include LICENSE
 include package.json
+include requires-install.txt
+include requires-dev.txt
+exclude dash_mantine_components/metadata.json
```

Comment:
This works, but it's a little funny to include `*.json` above and then exclude this one. The result of those two together is just to include `dash_mantine_components/package-info.json` so it would be cleaner to just include that file by itself.

While we're at it, it's also redundant (but common across most dash component suites) that we include both `package.json` and `dash_mantine_components/package-info.json` since these are identical (`package.json` is copied to `dash_mantine_components/package-info.json` during build). The former is referenced in `setup.py` to pull information into the pypi record, the latter in `dash_mantine_components/__init__.py` to get the package name and version, and I believe that's it. This file isn't very big, but we should be able to remove `package.json` entirely if we point `setup.py` to the other one.